analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -575,8 +575,8 @@ class Pub(
     private fun commandPub(): String = "${command()} pub"
 
     private fun commandFlutter(): String =
-        if (flutterAbsolutePath.isDirectory) "$flutterAbsolutePath${File.separator}$flutterCommand packages"
-        else "$flutterCommand packages"
+        if (flutterAbsolutePath.isDirectory) "$flutterAbsolutePath${File.separator}$flutterCommand pub"
```

Comment:
Commit message: No dot at the end of the title.

More importantly, was "packages" renamed to "pub" already in Flutter version 2.2.3-stable, which is the version we bootstrap by default? If yes, why did the test work until now? If no, shouldn't we stick to "packages" for older version of Flutter, i.e. decide base on the Flutter version in use whether to use "packages" or "pub"?

Reply:
> was "packages" renamed to "pub" already in Flutter version 2.2.3-stable, which is the version we bootstrap by default?

Yes.

> If yes, why did the test work until now?

Because `flutterCommand()` / `flutter pub get` was never really used. Also the test called just `dart pub get` (via `run()`).
This does work in _some_ cases, but not in cases like I outlined in https://github.com/oss-review-toolkit/ort/issues/4651.

Reply:
So what if users decide to bootstrap an older version of Flutter than the one we bootstrap by default? Should we support that case by still deciding based on the Flutter version which command to call?

Reply:
I downloaded Flutter 1.0.0 from 04.12.2018 to check. This version also includes `flutter pub` and the important `get` sub-command. So I think we should be completely backwards compatible.